public/javascripts/SVValidate/src/Main.js

```diff
@@ -10,8 +10,8 @@ var svv = svv || {};
 function Main (param) {
     svv.canvasHeight = param.canvasHeight;
     svv.canvasWidth = param.canvasWidth;
-
     svv.missionsCompleted = param.missionSetProgress;
+    svv.validationsCompleted = param.completedValidations;
```

Comment:
Instead of saving this in a global variable (`svv.validationsCompleted`) you should pass in param.completedValidations to the StatusField constructor and save the variable locally in the StatusField object.